apps/classifications/signals.py

```diff
@@ -21,13 +22,29 @@ def get_ai_classification(sender, instance, **kwargs):
                 settings.AI_API_AUTH_TOKEN:
             try:
                 response = call_ai_api(comment=instance)
-                if response.status_code == 200 \
-                        and response.json()['classification'] == 'OFFENSE':
+                if response.status_code == 200:
+                    # version 2
+                    if ai_api_version == '2.0' \
+                            and response.json()['classification'] == 'OFFENSE':
+
+                        classification = AIClassification(
+                            comment=instance,
+                            classifications=['OFFENSIVE']
+                        )
+                        classification.save()
+                    # version 3
+                    elif ai_api_version == '3.0' and \
```

Comment:
we could also cast the string into a number and say 
```
if version <= 2:
    do this
else:
    do that
```
But as we don't really want to use versions <= 2.0, it's also fine like that and remove the whole if after the next release

Reply:
Yes, I think we can remove that soon, so left it like that!